lib/cli/src/commands/publish.rs

```diff
@@ -34,13 +34,14 @@ pub struct Publish {
     ///
     /// Note that this is not the timeout for the entire publish process, but
     /// for each individual query to the registry during the publish flow.
-    #[clap(long, default_value = "30s")]
+    #[clap(long, default_value = "2m")]
```

Comment:
I am increasing this for all the publish queries because 20 seconds is too small anyway.